Source/DOM classes/Core DOM/Comment.h

```diff
@@ -11,7 +11,13 @@
 
 #import "CharacterData.h"
 
```

Comment:
Ugly fix. I found that MaddTheSane change all the class in `Core DOM` with the `SVG` prefix, for example, `Attr -> SVGKAttr`, `Node -> SVGKNode`. I think it's a good idea and can solve many potential future issue. 

Since the user will not always directlly use class in `Core DOM`, I think renaming maybe adoptable. What do you think.

Reply:
No-one has invented a satisfactory solution to this problem - ObjectiveC is too weak a language to solve it neatly.

I believe the best solution is:

1. Create a new open-source project "CoreDOM"
2. Move all the DOM source code to that project
3. Add the CoreDOM project as one of SVGKit's dependencies

...then anyone is free to re-implement the public headers of CoreDOM and replace it.

(everything else people have suggested or thought of and tried ... hasn't worked. ObjectiveC just doesn't support dynamic linking, late binding, replaceable vendor code, etc)

NB: there may be a reason that even this solution doesn't work -- I can't remember if anyone tried it :) -- but I believe the only reason no-one did it was that it felt like a lot of hassle to setup an extra project in github?

Reply:
MaddTheSane's solution is really not good. It prevents anyone from ever replacing the CoreDOM implementation (most of which I hacked out in a few days, just implementing the spec as simply as possible, and probably could be greatly optimized / expanded)

Reply:
@adamgit You means some advanced user, may provide a custom implementation of DOM object used in our repo ?

Maybe I think the better way, it to define all set of protocol in Objective-C. For example, we can define `@protocol Document @end` as well as `@interface Document @end`. Then people who use the custom implementation can replace the class of we used.

However, it's a big changes. So I think they can just fork the repo and use another same class name, but changing to different implementation instead. If they'd like this approach, I think move the Core DOM code into another repo is acceptable. Because they can focus on DOM spec but not SVG parser or something non-related thing.

Reply:
TL;DR: unless someone moves the DOM implementation into a new repo, I prefer leaving it with the current names (that were defined by the official W3C standard! They are supposed to be the same in every platform, every programming language. That's explicitly how the spec was written - and is why they look strange on some platforms)